sdk/eventhub/event-hubs/samples/javascript/README.md

```diff
@@ -18,8 +18,9 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | [useWithIotHub.js][usewithiothub]                                         | demonstrates how to use the `EventHubClient` with an `IotHub` instance                                                                                                                  |
 | [websockets.js][websockets]                                               | demonstrates how to use WebSockets enable Event Hubs to work over an HTTP proxy and in environments where the standard AMQP port 5671 is blocked (see the sample for more information)  |
 | [usingAadAuth.js][usingaadauth]                                           | demonstrates how to instantiate EventHubsClient using AAD token credentials obtained from using Service Principal Secrets                                                               |
-| [receiveEventsUsingCheckpointStore.js][receiveeventsusingcheckpointstore] | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance (requires running the `sendEvents` sample first) |
-| [receiveEvents.js][receiveevents]                                         | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance, as well as checkpointing along the way.         |
+| [receiveEvents.js][receiveevents]                                         | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance.                                                 |
+
+**Note**: `EventHubConsumerClient` supports checkpointing using the `@azure/eventhubs-checkpointstore-blob` and `@azure/storage-blob` packages. For an example that demonstrates how to use this functionality, see [this link][checkpointing].
```

Comment:
Link texts should have contextual information to help screenreaders

```suggestion
**Note**: `EventHubConsumerClient` supports checkpointing using the `@azure/eventhubs-checkpointstore-blob` and `@azure/storage-blob` packages. For an example that demonstrates how to use this functionality, see the sample [receiveEventsUsingCheckpointStore.js][checkpointing].
```

Reply:
Addressed this in the latest commit